src/relayserver/TransactionManager.ts

```diff
@@ -92,14 +94,13 @@ export class TransactionManager {
     } finally {
       releaseMutex()
     }
-    const receipt = await this.contractInteractor.sendSignedTransaction(signedTx)
-    log.info('\ntxhash is', receipt.transactionHash)
-    if (receipt.transactionHash.toLowerCase() !== storedTx.txId.toLowerCase()) {
-      throw new Error(`txhash mismatch: from receipt: ${receipt.transactionHash} from txstore:${storedTx.txId}`)
+    const transactionHash = await this.contractInteractor.broadcastTransaction(signedTx)
```

Comment:
why id you rename it? it IS sendSignedTransaction - the way it shoudl be ..

Reply:
To differentiate from web3 sendSignedTransaction that locks thread until mined. "Broadcast" implies not caring about it being delivered IMO.